src/officehours_api/models.py

```diff
@@ -183,12 +184,17 @@ def delete(self, *args, **kwargs):
         super().delete(*args, **kwargs)
 
     @property
-    def line_place(self):
+    def line_place(self) -> Optional[int]:
         if not self.queue:
             return None
         meetings = self.queue.meeting_set.order_by('id')
-        for i in range(0, len(meetings)):
-            m = meetings[i]
+        unstarted_meetings: List[Meeting] = [
+            meeting for meeting in meetings if meeting.status != MeetingStatus.STARTED
+        ]
+        if self not in unstarted_meetings:
+            return None
```

Comment:
It looks like these two lines wouldn't do anything except iterate through the list an extra time and removing them would retain the same behavior, since methods return `None` by default (i.e. when no `return` is specified). They do have the value of making it perfectly explicit what happens when `self` isn't there, which is good, and these lists would rarely be long, so there is probably no noticeable performance implication to iterating twice. Just something to be aware of.

Reply:
Nice point, @jlost. If we want to be explicit but not iterate an extra time, could I delete these lines and put `return None` after the loop that was already there?

Reply:
Yup, absolutely

Reply:
Cool, done.